core/pom.xml

```diff
@@ -168,6 +187,18 @@
 
         <dependencies>
 
+            <dependency>
+                <groupId>com.xatkit</groupId>
+                <artifactId>chat-platform-runtime</artifactId>
+                <version>3.0.1-SNAPSHOT</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.xatkit</groupId>
+                <artifactId>react-platform-runtime</artifactId>
+                <version>4.0.1-SNAPSHOT</version>
+            </dependency>
+
```

Comment:
Remove dependencies to specific platforms in Xatkit Runtime.
I'll take care of it.

Reply:
Sorry, I pushed this by mistake.